### ISSUE TITLE
different URL for different documentation

### ISSUE THREAD

**Issue by davydden** [dealii/dealii#6595]
It's really annoying that https://www.dealii.org/9.0.0/index.html is used both for the outline / TOC like page and **some** subsection (when you click on ``Testsuite'', for example). Not only it prevents you from giving direct links to users, but it's also bad for Search Engine Optimization (SEO).
These should really be different pages with different URLs.

p.s. while there `Technical reports` is broken.

**Comment by bangerth** [comment]
Yes, HTML frames was a thing of the 1990s, wasn't it? I typically right click on one of the menu items and open them in a separate tab -- that gives me a sharable link.

**Comment by davydden** [comment]
> Yes, HTML frames was a thing of the 1990s, wasn't it?

yeap.

while there, let me point out a another thing about the website:

`9.0.0 -> README` / `9.0.0->DOCUMENTATION` / `info->development`, `dev->README`, `dev-documentation`, `dev-Testing info` should keep the header from `dealii.org` and general style (colours, fonts, background, etc) so that users can quickly navigate where then need. Currently they are disjoint pages in terms of web-design (in our case IMO it's acceptable to have `tutorials` and `manual` as a fully separate designs as those are generated by doxygen). 

Bottom line is that there is really no need to have a caledoscope of layouts, we just need 

(i) the layout from the main page 
(ii) doxygen generated documentation (manual, tutorials, examples, etc).

**Comment by kronbichler** [comment]
Since we're discussing the homepage right now, one very annoying thing I noticed: Our front page dealii.org renders very badly on Android mobile phones with the Chromium browser (at least for me). The problem appears to be the orange box "Online course on Finite Element Analysis ..." which does not break lines. As a consequence, the drop-down menu is shifted to outside the screen area and one needs to scroll left and right all the time to use it. Can we make this course advertisement a more standard link? Furthermore, I would move this advertisement to a less prominent position of our page - it's neither new nor that important for our project in my opinion, isn't it? I can try to fix that if there is consensus about this, even though I must admit I'm not a CSS expert.

**Comment by bangerth** [comment]
I don't even know whether the course is still running. But either way, we could move it into the same menu where we also have the links to my videos. Would that help?

**Comment by bangerth** [comment]
Yes, good strategy. Frames were originally a way to do what you suggest doing with cmake (namely, not duplicate the same thing in multiple files), but they have not stood the time. Using cmake is one simple way. Using javascript would be another, in order to load parts of a page from different files. (The third option are server-side includes, but that's harder to administer, and does not work well when you want to look at documentation locally.)
